mkdocs_multirepo_plugin/structure.py

```diff
@@ -30,12 +30,17 @@ def resolve_nav_paths(nav: List[Dict], section_name: str) -> None:
     """Adds the section_name to the beginning of all paths in a MkDocs nav object"""
     for index, entry in enumerate(nav):
         if isinstance(entry, str):
-            nav[index] = str(section_name / Path(entry)).replace("\\", "/")
+            if entry.startswith("http"):

```

Comment:
And ftp or other schemas? What about using `urlparse`?

Reply:
also, you can make this one line.

```python
nav[index] = str(section_name / Path(entry)).replace("\\", "/") if not urlparse.some_method(entry) else entry
```